tarantool_test.go

```diff
@@ -693,6 +694,514 @@ func TestClient(t *testing.T) {
+					resp.MetaData.ColumnsInfo[j].FieldIsNullable, i)
+			}
+			if resp.MetaData.ColumnsInfo[j].FieldCollation !=
+				responses[i].MetaData.ColumnsInfo[j].FieldCollation {
+				t.Errorf("Response MetaData is wrong after Execute\nExpected: %v Got: %v\nQuery number: %d",
+					responses[i].MetaData.ColumnsInfo[j].FieldCollation,
+					resp.MetaData.ColumnsInfo[j].FieldCollation, i)
+			}
+			if resp.MetaData.ColumnsInfo[j].FieldName !=
+				responses[i].MetaData.ColumnsInfo[j].FieldName {
+				t.Errorf("Response MetaData is wrong after Execute\nExpected: %v Got: %v\nQuery number: %d",
+					responses[i].MetaData.ColumnsInfo[j].FieldName,
+					resp.MetaData.ColumnsInfo[j].FieldName, i)
+			}
+			if resp.MetaData.ColumnsInfo[j].FieldSpan !=
+				responses[i].MetaData.ColumnsInfo[j].FieldSpan {
+				t.Errorf("Response MetaData is wrong after Execute\nExpected: %v Got: %v\nQuery number: %d",
+					responses[i].MetaData.ColumnsInfo[j].FieldSpan,
+					resp.MetaData.ColumnsInfo[j].FieldSpan, i)
+			}
+			if resp.MetaData.ColumnsInfo[j].FieldType !=
+				responses[i].MetaData.ColumnsInfo[j].FieldType {
+				t.Errorf("Response MetaData is wrong after Execute\nExpected: %v Got: %v\nQuery number: %d",
+					responses[i].MetaData.ColumnsInfo[j].FieldType,
+					resp.MetaData.ColumnsInfo[j].FieldType, i)
+			}
+		}
+	}
+}
+
+func TestSQLBindings(t *testing.T) {
+	// Data for test table
+	testData := map[int]string{
+		1: "test",
+	}
+
+	// Check for skip SQL tests if tarantool version < 2.0.0
+	isLess, err := test_helpers.IsTarantoolVersionLess(2, 0, 0)
+	if err != nil {
+		t.Errorf("Could not check the Tarantool version")
+		return
+	}
+	if isLess {
+		return
+	}
+
+	var resp *Response
+	var conn *Connection
+
+	conn, err = Connect(server, opts)
+	if err != nil {
+		t.Errorf("Failed to connect: %s", err.Error())
+		return
+	}
+	if conn == nil {
+		t.Errorf("conn is nil after Connect")
+		return
+	}
+	defer conn.Close()
+
+	resp, err = conn.Execute(createTableQuery, []interface{}{})
+	if err != nil {
+		t.Errorf("Failed to Execute: %s", err.Error())
+	}
+	if resp == nil {
+		t.Errorf("Response is nil after Execute")
+	}
+	if resp.Code != 0 {
+		t.Errorf("Failed to Execute: %d", resp.Code)
+	}
+	if resp.SQLInfo.AffectedCount != 1 {
+		t.Errorf("Incorrect count of created spaces: %d", resp.SQLInfo.AffectedCount)
+	}
+
+	resp, err = conn.Execute(insertQuery, []interface{}{1, testData[1]})
+	if err != nil {
+		t.Errorf("Failed to Execute: %s", err.Error())
+	}
+	if resp == nil {
+		t.Errorf("Response is nil after Execute")
+	}
+	if resp.Code != 0 {
+		t.Errorf("Failed to Execute: %d", resp.Code)
+	}
+	if resp.SQLInfo.AffectedCount != 1 {
+		t.Errorf("Incorrect count of affected rows: %d", resp.SQLInfo.AffectedCount)
+	}
+
+	// test all types of supported bindings
+	// prepare named sql bind
+	sqlBind := map[string]interface{}{
+		"id":   1,
+		"name": "test",
+	}
+	sqlBind2 := struct {
+		Id   int
+		Name string
+	}{1, "test"}
+
```

Comment:
Newlines are a bit weird: there is one between `sqlBind2` and `sqlBind3` and no newlines between `sqlBind` and `sqlBind2` or `sqlBind3` and `namedSQLBinds`.

Reply:
Fixed.